src/test/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesTest.scala

```diff
@@ -277,6 +278,51 @@ class KafkaConsumerPropertiesTest extends AnyFunSuite with BeforeAndAfterEach wi
     assert(BaseProperties(properties).getSSLEndpointIdentificationAlgorithm() === "https")
   }
 
+  test("getRecordKeyFormat returns user provided property value") {
+    properties = Map("RECORD_KEY_FORMAT" -> "keyFormat")
+    assert(BaseProperties(properties).getRecordKeyFormat() === "keyformat")
+  }
+
+  test("getRecordKeyFormat returns default value if not set") {
+    assert(BaseProperties(properties).getRecordKeyFormat() === "string")
+  }
+
+  test("getRecordValueFormat returns user provided property value") {
+    Seq("avro", "JsOn", "String").foreach { value =>
+      properties = Map("RECORD_VALUE_FORMAT" -> value)
+      assert(BaseProperties(properties).getRecordValueFormat() === value.toLowerCase(ENGLISH))
```

Comment:
here you use the same function in the test then in the prodction code. Maybe it would make sense to add the values directly in lower case here. On the other hand, we can probably assume that it works correctly...

Reply:
Good idea! Changed.